pyndl/ndl_parallel.pyx

```diff
@@ -52,29 +52,39 @@ def learn_inplace(binary_file_paths, np.ndarray[dtype_t, ndim=2] weights,
     cdef unsigned int length_all_outcomes = all_outcomes.shape[0]
     cdef char* fname
     cdef unsigned int start_val, end_val, ii, number_parts
-    cdef int error = 4
+    cdef int error = 3
+    # error codes:
```

Comment:
would be great, if we could use constants / enum for the error codes, this  would improve readability